blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -68,6 +68,10 @@ home.pipelineslist.header.name=Name
 home.pipelineslist.header.pullrequests=PR
 home.pipelineslist.row.failing={0} failing
 home.pipelineslist.row.passing={0} passing
+home.placeholder.linkhref=/create-pipeline
```

Comment:
I agree that internal routes should `not` be in the i18n bundles. I will fix this in the next empty state PR.

For external URL's to Jenkins docs, is it fair to leave those in? It seems like those might get i18n's within a reasonable time frame, and it does seem like a "UI string" that someone might want to change easily without mucking through the React code.